exporter.go

```diff
@@ -289,8 +303,27 @@ func main() {
 
 	registry.MustRegister(buildInfoMetric)
 
+	sslConfig := &tls.Config{InsecureSkipVerify: !*sslVerify}
+	if *sslCaCert != "" {
+		caCert, err := ioutil.ReadFile(*sslCaCert)
+		if err != nil {
+			log.Fatalf("Loading CA cert failed: %v", err)
+		}
+		sslCaCertPool := x509.NewCertPool()
+		sslCaCertPool.AppendCertsFromPEM(caCert)
```

Comment:
`AppendCertsFromPEM` function reports whether any certificates were successfully parsed in a boolean returned argument. Could you possible check that result and fatal with an error if there is a problem?